Loader/Program.cs

```diff
@@ -140,7 +130,20 @@ static void Main(string[] args)
 				File.WriteAllText(Path.Combine(outputRoot, pair.Key.ToLower() + "-items.json"), JsonConvert.SerializeObject(pair.Value));
 			}
 
+			// Ships and ground vehicles
```

Comment:
What's the reason for moving the ship loading lower down?

Reply:
It was because the "Load items" part need ammoIndex, manufacturerIndex. it's to keep the three together.